zboxcore/fileref/fileref.go

```diff
@@ -57,7 +57,7 @@ func (a *Attributes) Validate() (err error) {
 }
 
 type FileRef struct {
-	Ref                 `json:",squash"`
+	Ref
```

Comment:
I would suggest to leave `squash` there. Because we don't know if it is default on embedded struct in all mapstructure versions.

Reply:
I checked with json tags and there is no squash option.
For mapstructure, it should have mapstructure in place of json, right?

Reply:
I think it is designed for mapstructure, not for json. so please double check it on validator.

Reply:
I would suggest to leave it if we can know how it works on old code. I don't know all old code as well .